### ISSUE TITLE
Updated readme for putting large files on box?

### ISSUE THREAD

**Issue by sosey** [spacetelescope/jwst-scsb#16]
For better access for all users, I dont think box was an option when this repo was first started

**Comment by pllim** [comment]
Yes, I think that is consistent with https://github.com/spacetelescope/style-guides/blob/master/guides/where-to-put-your-data.md

**Comment by sosey** [comment]
archiving repo
